context/app/static/js/shared-styles/buttons/index.js

```diff
@@ -29,7 +29,14 @@ const WhiteBackgroundToggleButton = styled(ToggleButton)`
   }
 `;
 
-function TooltipToggleButton({ children, tooltipComponent, tooltipTitle, buttonComponent, id, ...rest }) {
+function TooltipToggleButton({
+  children,
+  tooltipComponent,
+  tooltipTitle,
+  buttonComponent: _buttonComponent,
```

Comment:
Same as in the other case; if this is ever called with a `buttonComponent` prop defined, it would be implicitly propagated to `WhiteBackgroundToggleButton`, so I marked it as unused with the underscore prefix to be safe.